Sources/Build/Triple.swift

```diff
@@ -128,7 +132,10 @@ public struct Triple: Encodable {
     public static let s390xLinux = try! Triple("s390x-unknown-linux")
     public static let arm64Linux = try! Triple("aarch64-unknown-linux")
     public static let armLinux = try! Triple("armv7-unknown-linux-gnueabihf")
-    public static let android = try! Triple("armv7-unknown-linux-androideabi")
+    public static let armAndroid = try! Triple("armv7-unknown-linux-androideabi")
```

Comment:
One last thing, the [target triple](https://developer.android.com/ndk/guides/other_build_systems) should be `armv7a` (sorry I missed that before)

Reply:
Will do.